### ISSUE TITLE
Replace the binary Wsk with REST interface WskRest

### ISSUE THREAD

**Issue by houshengbo** [ibm-functions/runtime-nodejs#69]
All the non-CLI related tests should sunset the wsk, and re-implement them with wskrest.

```
import common.wsk 
val wsk = new Wsk
```

need to be changed to

```
import common.rest.WskRest
val wsk = new WskRest
```

plus there may be changes to the rest of the tests as well, since the response result may have been changed.

**Comment by houshengbo** [comment]
WskBasicIBMNodeTests
IBMNodeJsActionCloudantTests
IBMNodeJsActionDB2Tests
IBMNodeJsActionWatsonTests

CredentialsIBMNodeJsActionWatsonTests
CredentialsIBMNodeJsCOSTests
CredentialsIBMNodeJsDb2CloudTests
CredentialsIBMNodeJsActionCloudantTests

**Comment by csantanapr** [comment]
fix in #74
